week-3/Homework/mandatory/4-eligible-students.js

```diff
@@ -7,8 +7,12 @@
   - Returns an array containing onl the names of the who have attended AT LEAST 8 classes
 */
 
-function eligibleStudents() {
-
+function eligibleStudents(students) {
+  let studentsName = students.filter(([a, b]) => b >= 8);
+  let filterStudentsName = studentsName.map(x => x.slice(0, 1));
+  let finalNames = [].concat(...filterStudentsName);
```

Comment:
This is interesting 🙂Could you explain why this line is necessary?

Reply:
because I have an 2d array and I nedeed to make 1d  so I concatinated an empty array with my 2d array